### PR TITLE
Justin/add gitignore

### PR DESCRIPTION
**Pull request by JSprecco**
Added the .gitignore and json files to complete the initial repo. Also made a mistake on the README and reverted it to the original.

### CODE REVIEW

**Review by joebarteluce — APPROVED** [review]

File: package-lock.json
Comment (Da-Scher) [inline package-lock.json:1]: I was wrong, package-lock.json  we should add the file with 12K lines to the .gitignore

**Review by Da-Scher — CHANGES_REQUESTED** [review]
Check the review for my comments on the `package-lock.json` file.
Steps to get this PR approved:
- [x ] ~~add `package-lock.json` to the file `.gitignore` to your local repo.~~
- [x ] ~~use the command `git rm .package-lock.json`~~
- [x] if you have `package-lock.json` in the `.gitignore`, remove `package-lock.json` from `.gitignore`
- [x] `npm audit` to check for vulnerable packages
- [x] `npm audit fix` to update vulnerable versions
- [x] `git add` the changes
- [x] `git push` to your branch

File: package-lock.json
Comment (JSprecco) [reply package-lock.json:1]: It may be large but having it in the repo will gaurentee the dependencies are met on all of our individual builds.

Comment (Da-Scher) [reply package-lock.json:1]: On further inspection, you are correct that package-lock.json is necessary even at its size. I will update the requirements for the merge to address the '2 high risk vulnerabilities' with Vite.

Comment (JSprecco) [reply package-lock.json:1]: I'm getting 0 vulnerabilities on my end.

**Pull request merged**